pkgs/development/interpreters/yex-lang/derivation.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, pkgconfig, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "yex-lang-${version}";
```

Comment:
Can we use pname + version (see #103997)?

Reply:
Done